### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by marciantemark**
The draft charter has been created and ready for offline discussions

### CODE REVIEW

File: CHANGE_MANAGEMENT.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Overview
+## **What is the Digital Quality Implementer's Community?**
+DQIC (pronounced "D-Quick") is a meritocratic, consensus-based community project focused on Digital Quality measures. Anyone with interest in the project can participate as a Stakeholder and a Contributor. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:3]: Can we say "focused on Digital Quality Improvement applications"? I think there was agreement that this is broader than just measures

File: CHANGE_MANAGEMENT.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+# Overview
+## **What is the Digital Quality Implementer's Community?**
+DQIC (pronounced "D-Quick") is a meritocratic, consensus-based community project focused on Digital Quality measures. Anyone with interest in the project can participate as a Stakeholder and a Contributor. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community. 
+
+## **Who can be part of DQIC?**
+Any interested stakeholder that is involved in digital quality measurements is welcome; however, participation as a committer or DQIC Steering Comittee (DSC) member requires active participation and contributions as Committers and DSC members, compliance with the DQIC charter, and a monthly membership fee. Contact DQIC’s Executive Committee (https://www.ncqa.org/digital-quality-implementers-community/) to learn more about official membership in DQIC . All products produced by DQIC will be open-source and available to the public.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:6]: "digital quality improvement"

File: CHANGE_MANAGEMENT.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- Providing moral support (a ‘thank you’ goes a long way)
+- Stakeholders who continue to engage with the project and its community will often become more and more involved. Such stakeholders may find themselves becoming contributors, as described in the next section.
+
+## Contributors
+
+Contributors are community members who contribute in concrete ways to DQIC. Anyone can become a contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements and no selection process.
+
+In addition to their actions as stakeholders, contributors may also find themselves doing one or more of the following:
+
+- Contributing to the [Cinical Quality Language](https://github.com/cqframework/clinical_quality_language/tree/master) project
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:32]: This should be [CQL Tests Repository](https://github.com/cqframework/cql-tests) (this repository), right?

File: CHANGE_MANAGEMENT.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+- Reporting bugs
+- Identifying requirements
+- Providing graphics and web design
+- Programming
+- Assisting with project infrastructure
+- Writing documentation
+- Fixing bugs
+- Adding features
+
+In addition, it is highly encouraged to be active Contributors engage with the project through the issue tracker and Zulip, or by writing or editing documentation. They submit changes to the project itself via pull requests, which will be considered for inclusion in the project by existing committers (see next section). Use of Zulip is the most appropriate place to ask for help when making that first contribution.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:43]:
```suggestion
In addition, Contributors are encouraged to engage with the project by [submitting issues](https://github.com/cqframework/cql-tests/issues) and [the CQL stream in the FHIR Zulip chat](https://chat.fhir.org/#narrow/stream/179220-cql), or by writing or editing documentation. Contributors can submit changes to the project itself via pull requests, which will be considered for inclusion in the project by existing committers (see next section). Use of Zulip is the most appropriate place to ask for help when making that first contribution.
```

File: CHANGE_MANAGEMENT.md (hunk 3 above)
Comment (brynrhodes) [reply CHANGE_MANAGEMENT.md:32]:
```suggestion
- Contributing to the [CQL Tests](https://github.com/cqframework/cql-tests) repository (this project)
```

File: CHANGE_MANAGEMENT.md (hunk 2 above)
Comment (brynrhodes) [reply CHANGE_MANAGEMENT.md:6]:
```suggestion
Any interested stakeholder that is involved in digital quality improvement is welcome; however, participation as a committer or DQIC Steering Comittee (DSC) member requires active participation and contributions as Committers and DSC members, compliance with the DQIC charter, and a monthly membership fee. Contact DQIC’s Executive Committee (https://www.ncqa.org/digital-quality-implementers-community/) to learn more about official membership in DQIC . All products produced by DQIC will be open-source and available to the public.
```

File: CHANGE_MANAGEMENT.md (hunk 1 above)
Comment (brynrhodes) [reply CHANGE_MANAGEMENT.md:3]:
```suggestion
DQIC (pronounced "D-Quick") is a meritocratic, consensus-based community project focused on Digital Quality improvement applications. Anyone with interest in the project can participate as a Stakeholder and a Contributor. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community. 
```

File: CHANGE_MANAGEMENT.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+## Voting
+
+Not all decisions can be made using the process of consensus. Issues such as those affecting the strategic direction, project releases, or legal standing of the project must gain explicit approval in the form of a vote. Every member of the community is encouraged to express their opinions in all discussion and all votes. However, only project committers and/or DSC members (as defined above) have binding votes for the purposes of decision making.
+
+## Contributions
+
+The project uses a stable-trunk methodology, meaning that the master branch must be always kept in a releasable state. This is ensured through regression tests and continuous integration is used to check pull requests to the master branch.
+
+Only trivial commits (such as documentation changes, typos, and packaging changes) can be made directly to the master branch. All other changes to the master branch must be made using a pull request and must be reviewed prior to being applied.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:109]:
```suggestion
All changes to the main branch must be made using a pull request and must be reviewed by at least two committers that were not the author of the pull request prior to being applied.
```

File: CHANGE_MANAGEMENT.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+Changes must be reviewed by at least two committers. Large, significant, or breaking changes must have full committer consensus before being applied.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:111]:
```suggestion
In addition, large, significant, or breaking changes must have full committer consensus before being applied.
```

File: CHANGE_MANAGEMENT.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+In general, when a new version of the CQL specification is created, a new branch of the test repository is created to implement support for that version. Changes to this branch can be committed directly. Once the new version of the specification is published, the tests in this repository are added.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:113]:
```suggestion
In general, when a new version of the CQL specification is created, a new branch of the test repository is created to implement support for that version. Although changes to this new version branch may be committed directly, best practice is to use pull requests for the new version branch as well. Once the new version of the specification is published, the tests in this repository are added.
```

File: CHANGE_MANAGEMENT.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+Note that changes must always align with the CQL specification. Any changes to the CQL specification itself are governed by HL7 and must be done through the HL7 process, either by submitting an STU comment, or balloting a new version of the specification.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:115]:
```suggestion
Note that changes must always align with the CQL specification. Changes made to tests in this repository will be submitted back to the informative tests published as part of the CQL specification. Any changes to the CQL specification itself are governed by HL7 and must be done through the HL7 process, either by submitting an STU comment, or balloting a new version of the specification.
```

File: CHANGE_MANAGEMENT.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+## Communication Channels
+
+Communication should be done in an open and public manner. The project uses many different channels for open communication, including:
+
+- Chat
+- Email
+- Public Zulip
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:123]:
```suggestion
- Public [CQL Stream in the FHIR Zulip Chat](https://chat.fhir.org/#narrow/stream/179220-cql)
```

**Review by brynrhodes — CHANGES_REQUESTED** [review]

File: CHANGE_MANAGEMENT.md (hunk 1 above)
Comment (marciantemark) [reply CHANGE_MANAGEMENT.md]: Yes that makes a lot of sense.

File: CHANGE_MANAGEMENT.md (hunk 4 above)
Comment (marciantemark) [reply CHANGE_MANAGEMENT.md]: Much cleaner and like the links. Thanks!

**Review by marciantemark — COMMENTED** [review]
All changes reviewed and most approved

File: CHANGE_MANAGEMENT.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
+# Overview
+## **What is the Digital Quality Implementer's Community?**
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:2]:
```suggestion
```

File: CHANGE_MANAGEMENT.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+DQIC (pronounced "D-Quick") is a meritocratic, consensus-based community project focused on Digital Quality improvement applications. Anyone with interest in the project can participate as a Stakeholder and a Contributor. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:3]:
```suggestion
This is a meritocratic, consensus-based community project. Anyone with interest in the project can join the community, contribute to the project design and participate in the decision making process. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community.

Although these processes are adapted from the [OSSWatch Meritocratic Model](http://oss-watch.ac.uk/resources/meritocraticgovernancemodel), this documentation is a formalization of existing processes involving relevant stakeholders.

In particular, this project is a primary focus of the [Digital Quality Implementer's Community](https://www.ncqa.org/digital-quality-implementers-community/).
```

File: CHANGE_MANAGEMENT.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
+# Overview
+## **What is the Digital Quality Implementer's Community?**
+DQIC (pronounced "D-Quick") is a meritocratic, consensus-based community project focused on Digital Quality improvement applications. Anyone with interest in the project can participate as a Stakeholder and a Contributor. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community. 
+
+## **Who can be part of DQIC?**
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:5]:
```suggestion
```

File: CHANGE_MANAGEMENT.md [hunk 13]
```diff
@@ -0,0 +1,148 @@
...
+Any interested stakeholder that is involved in digital quality improvement is welcome; however, participation as a committer or DQIC Steering Comittee (DSC) member requires active participation and contributions as Committers and DSC members, compliance with the DQIC charter, and a monthly membership fee. Contact DQIC’s Executive Committee (https://www.ncqa.org/digital-quality-implementers-community/) to learn more about official membership in DQIC . All products produced by DQIC will be open-source and available to the public.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:6]:
```suggestion
```

File: CHANGE_MANAGEMENT.md [hunk 14]
```diff
@@ -0,0 +1,148 @@
...
+Although these processes are adapted from the [OSSWatch Meritocratic Model](http://oss-watch.ac.uk/resources/meritocraticgovernancemodel), this documentation is a formalization of existing processes involving relevant stakeholders.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:7]:
```suggestion
```

File: CHANGE_MANAGEMENT.md [hunk 15]
```diff
@@ -0,0 +1,148 @@
...
+
+# Roles
+ There are four roles in the DQIC.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:10]:
```suggestion 
```

File: CHANGE_MANAGEMENT.md [hunk 16]
```diff
@@ -0,0 +1,148 @@
...
+
+## **Who can be part of DQIC?**
+Any interested stakeholder that is involved in digital quality improvement is welcome; however, participation as a committer or DQIC Steering Comittee (DSC) member requires active participation and contributions as Committers and DSC members, compliance with the DQIC charter, and a monthly membership fee. Contact DQIC’s Executive Committee (https://www.ncqa.org/digital-quality-implementers-community/) to learn more about official membership in DQIC . All products produced by DQIC will be open-source and available to the public.
+Although these processes are adapted from the [OSSWatch Meritocratic Model](http://oss-watch.ac.uk/resources/meritocraticgovernancemodel), this documentation is a formalization of existing processes involving relevant stakeholders.
+
+# Roles
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:9]:
```suggestion
## Roles
```

File: CHANGE_MANAGEMENT.md [hunk 17]
```diff
@@ -0,0 +1,148 @@
...
+ There are four roles in the DQIC. 
+ 
+ ## Stakeholders
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:12]:
```suggestion
### Stakeholders
```

File: CHANGE_MANAGEMENT.md [hunk 18]
```diff
@@ -0,0 +1,148 @@
...
+Anyone can be a stakeholder; there are no special requirements; however, the most important stakeholders of this community consist of:
+- Members of government agencies that review and provide guidance as needed to support maturity of digital quality standards
+- Platform or technology owners that provide software to report on Digital Quality based on the Clinical Quality Language (CQL),
+- Payer and provider systems that use software to report on Digital Quality, and
+- Quality standard developers that maintain measures of quality in healthcare.
+
+Without these key stakeholders, the community would have limited value. Stakeholder participation is on a voluntary basis, but DQIC requests stakeholders to actively participate in the project and community as much as possible. Stakeholder contributions enable DQIC to ensure that they are satisfying the requirements of those stakeholders. Common stakeholder contributions include (but are not limited to):
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:19]:
```suggestion
Without these key stakeholders, the community would have limited value. Stakeholder participation is on a voluntary basis, but we encourage stakeholders to actively participate in the project and community as much as possible. Stakeholder contributions enable us to ensure that the project is satisfying the requirements of those stakeholders. Common stakeholder contributions include (but are not limited to):
```

File: CHANGE_MANAGEMENT.md [hunk 19]
```diff
@@ -0,0 +1,148 @@
...
+
+As contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated to become committers.
+
+## Committers
+
+Committers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. New committers can be nominated by any existing committer. Once they have been nominated, there will be a vote by the DQIC Steering Committee (DSC; see below). Committer voting is one of the few activities that takes place on the private management list. This is to allow DSC members to freely express their opinions about a nominee without causing embarrassment. Once the vote has been held, the aggregated voting results are published on the public mailing list. The nominee is entitled to request an explanation of any ‘no’ votes against them, regardless of the outcome of the vote. This explanation will be provided by the DSC Chair (see below) and will be anonymous and constructive in nature. <br> <br>
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:49]:
```suggestion
Committers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. New committers can be nominated by any existing committer. Once they have been nominated, there will be a vote by the Project Management Committee (PMC; see below). Committer voting is one of the few activities that takes place on the private management list. This is to allow PMC members to freely express their opinions about a nominee without causing embarrassment. Once the vote has been held, the aggregated voting results are published on the public mailing list. The nominee is entitled to request an explanation of any ‘no’ votes against them, regardless of the outcome of the vote. This explanation will be provided by the PMC Chair (see below) and will be anonymous and constructive in nature. <br> <br>
```

File: CHANGE_MANAGEMENT.md [hunk 20]
```diff
@@ -0,0 +1,148 @@
...
+&nbsp; &nbsp; **NOTE** As a committer, membership in the DQIC is required, which requires compliance with the DQIC charter, and a monthly membership fee. Contact [DQIC’s Executive Committee](https://www.ncqa.org/digital-quality-implementers-community/) to learn more about official membership in DQIC..
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:50]:
```suggestion
```

File: CHANGE_MANAGEMENT.md [hunk 21]
```diff
@@ -0,0 +1,148 @@
...
+- Quality standard developers that maintain measures of quality in healthcare.
+
+Without these key stakeholders, the community would have limited value. Stakeholder participation is on a voluntary basis, but DQIC requests stakeholders to actively participate in the project and community as much as possible. Stakeholder contributions enable DQIC to ensure that they are satisfying the requirements of those stakeholders. Common stakeholder contributions include (but are not limited to):
+
+- Evangelism of the project (e.g. a link on a website and word-of-mouth awareness)
+- Informing developers of strengths and weaknesses from a new stakeholder perspective
+- Providing moral support (a ‘thank you’ goes a long way)
+- Stakeholders who continue to engage with the project and its community will often become more and more involved. Such stakeholders may find themselves becoming contributors, as described in the next section.
+
+## Contributors
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:26]:
```suggestion
### Contributors
```

File: CHANGE_MANAGEMENT.md [hunk 22]
```diff
@@ -0,0 +1,148 @@
...
+- Assisting with project infrastructure
+- Writing documentation
+- Fixing bugs
+- Adding features
+
+In addition, Contributors are encouraged to engage with the project by [submitting issues](https://github.com/cqframework/cql-tests/issues) and [the CQL stream in the FHIR Zulip chat](https://chat.fhir.org/#narrow/stream/179220-cql), or by writing or editing documentation. Contributors can submit changes to the project itself via pull requests, which will be considered for inclusion in the project by existing committers (see next section). Use of Zulip is the most appropriate place to ask for help when making that first contribution.
+
+As contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated to become committers.
+
+## Committers
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:47]:
```suggestion
### Committers
```

File: CHANGE_MANAGEMENT.md [hunk 23]
```diff
@@ -0,0 +1,148 @@
...
+
+It is important to recognize that commitership is a privilege, not a right. That privilege must be earned and once earned it can be removed by the DSC (see next section) in extreme circumstances. However, under normal circumstances committership exists for as long as the committer wishes to continue engaging with the community.
+
+Committers are managed as members of the Committers Team in this repository.
+
+A committer who shows an above-average level of contribution to the project, particularly with respect to its strategic direction and long-term health, may be nominated to become a member of the DSC. This role is described below.
+
+## Digital Quality Implementers’ Community Steering Committee (DSC)
+
+The DSC consists of those individuals identified as ‘owners’ on the development site. The DSC is responsible for ensuring the smooth running of the project, including strategic planning, release cycles, change management, and changes to project governance. In addition, the DSC will ensure any changes needed to CQL are funneled to the HL7 CQL Workgroup for appropriate balloting and changes.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:60]:
```suggestion
The PMC consists of those individuals identified as ‘owners’ on the development site. The PMC is responsible for ensuring the smooth running of the project, including strategic planning, release cycles, change management, and changes to project governance. In addition, the PMC will ensure any changes needed to CQL are proposed to the HL7 CQL Workgroup for appropriate balloting and changes.
```

File: CHANGE_MANAGEMENT.md [hunk 24]
```diff
@@ -0,0 +1,148 @@
...
+
+Membership of the DSC is by invitation from the existing DSC members. A nomination will result in discussion and then a vote by the existing DSC members. DSC membership votes are subject to consensus approval of the current DSC members. DSC members must also be approved by the DSC Executive Committee.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:62]:
```suggestion
Membership of the PMC is by invitation from the existing PMC members. A nomination will result in discussion and then a vote by the existing PMC members. PMC membership votes are subject to consensus approval of the current PMC members.
```

File: CHANGE_MANAGEMENT.md [hunk 25]
```diff
@@ -0,0 +1,148 @@
...
+
+The DSC will be responsible for referring items that have consensus to the HL7 CQL Work Group for consideration.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:64]:
```suggestion
The PMC will be responsible for referring items that have consensus to the HL7 CQL Work Group for consideration.
```

File: CHANGE_MANAGEMENT.md [hunk 26]
```diff
@@ -0,0 +1,148 @@
...
+
+### DSC Chair
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:66]:
```suggestion
### PMC Chair
```

File: CHANGE_MANAGEMENT.md [hunk 27]
```diff
@@ -0,0 +1,148 @@
...
+
+The DSC Chair is a single individual, appointed by DQIC’s Executive Committee.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:68]:
```suggestion
The PMC Chair is a single individual, appointed by DQIC’s Executive Committee.
```

File: CHANGE_MANAGEMENT.md [hunk 28]
```diff
@@ -0,0 +1,148 @@
...
+
+The DSC Chair has no additional authority over other members of the DSC: the role is one of coordinator and facilitator. The Chair is also expected to ensure adherence to all governance processes and has the casting vote when the community fails to reach consensus.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:70]:
```suggestion
The PMC Chair has no additional authority over other members of the PMC: the role is one of coordinator and facilitator. The Chair is also expected to ensure adherence to all governance processes and has the casting vote when the community fails to reach consensus.
```

File: CHANGE_MANAGEMENT.md [hunk 29]
```diff
@@ -0,0 +1,148 @@
...
+
+### DSC Membership
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:72]:
```suggestion
#### PMC Membership
```

File: CHANGE_MANAGEMENT.md [hunk 30]
```diff
@@ -0,0 +1,148 @@
...
+- Email
+- Public [CQL Stream in the FHIR Zulip Chat](https://chat.fhir.org/#narrow/stream/179220-cql)
+
+Sometimes, communication occurs outside of these public channels, and that is okay; however, committers must summarize any private discussions that impact the tooling project in a public channel.
+
+## Code of Conduct
+
+In support of a healthy and inclusive community, we use and enforce a [code of conduct](https://github.com/cqframework/clinical_quality_language/blob/master/CODE_OF_CONDUCT.md) for all members of our community, including committers and DSC members. Our code of conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/).
+
+If you encounter any violation of these terms, please contact the DSC Chair or a DSC committee member. All reports will be kept in strict confidence and dealt with promptly.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:131]:
```suggestion
If you encounter any violation of these terms, please contact the PMC Chair or a PMC member. All reports will be kept in strict confidence and dealt with promptly.
```

File: CHANGE_MANAGEMENT.md [hunk 31]
```diff
@@ -0,0 +1,148 @@
...
+This project uses the Github Issues tracker to track all issues and feature requests. Anyone can submit an issue at any time. Committers are generally responsible for reviewing, responding to, and resolving issues in a timely manner.
+
+## Releases
+
+All packages within the project shall use semantic versioning. Any stakeholder can propose a release, but the DSC must review and approve the contents and timing of any release. Specifically, releases must be coordinated with impacted stakeholders and timed with availability of published versions of the specifications involved.<br>
+The DSC Chair is responsible for announcing releases to the community via the zulip chat. When appropriate, the DSC Chair should also announce plans for upcoming releases to solicit feedback from the community on release content and timing.
+
+# Change Management Continuous Improvement
+
+The DQIC is committed to continous improvement of code and process. While changes can happen at any time, the DSC and Committers shall be formally appreview and discuss governance bi-annually. As with all other artifacts, consensus shall be used to approve.
```
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md:144]:
```suggestion
This project is committed to continuous improvement of code and process. While changes can happen at any time, the PMC and Committers shall formally review and discuss governance bi-annually. As with all other artifacts, consensus shall be used to approve.
```

**Review by brynrhodes — CHANGES_REQUESTED** [review]

**Review by brynrhodes — APPROVED** [review]

**Pull request merged**